src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicSnippetsTransformer.java

```diff
@@ -134,14 +134,49 @@ private SnippetsFileView generateSnippets(GapicInterfaceContext context) {
       } else if (methodConfig.isPageStreaming()) {
         if (methodConfig.isFlattening()) {
           ImmutableList<FlatteningConfig> flatteningGroups = methodConfig.getFlatteningConfigs();
+          // Find flattenings that have ambiguous parameters, and mark them to use named parameters.
+          // These are page-stream flattenings that have one or two string parameters (that are not
+          // resource-names) than another flattening.
```

Comment:
Not sure what "than another" means here.

Reply:
I've tried to explain better.